changelog/fragments/autogenerated-makefile-bundle-prerequisite-task.yaml

```diff
@@ -0,0 +1,28 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Golang based operators, fix an error faced when the
+      [kustomize](https://kubernetes-sigs.github.io/kustomize/)
+      binary is not installed locally and the Makefile target
+      `bundle` is used by adding the kustomize target on it.
```

Comment:
```suggestion
      Added the `kustomize` make dependency to the `bundle` target scaffolded
      for Golang projects to install `kustomize` before running.
```

Reply:
:+1: changed. However, take into account that the previous text was suggested by @camilamacedo86, so, whatever you prefer.

Reply:
I thought that we had defined the standard of `For Golang based operators, For Helm based operators and etc ...`
However, it is fine 👍 I am ok with the new wording suggested by @estroz  as well.